### PR TITLE
Rename filter method "Assign to Me" to "Assigned to me"

### LINKED ISSUE #757: Overwritten change
Hi @sanbornsen ,

I've just discovered that you're commit 75ba0c66 has overwritten my already merged PR #744. I think it was due to renaming.

Please make sure that when you rename files (e.g. move to another location), you see it in the "git log --stat" message as a move.

Here's an example of such a message:

```
 src/tests/{app.component.spec.js => app.component.spec.js.2} | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
```

To achieve that it's a good habbit to 

1. `git mv oldfile newfile`
1. commit the changes `git commit -m "Moved oldfile to newfile"`

Only then start making changes to the file; **otherwise git cannot *always* track changes and movements**. 

Please, ensure that my changes are restored and also make sure that no other changes were overwritten.

### PR DESCRIPTION
**Pull request by sanbornsen**
Bringing back the changes from https://github.com/fabric8io/fabric8-planner/pull/744

Fixes #757

### CODE REVIEW

File: src/app/work-item/work-item.component.ts [hunk 1]
```diff
@@ -60,7 +60,7 @@ export class WorkItemComponent implements OnInit, AfterViewInit {
     if (this.loggedIn) {
       this.filters.push({
         id:  1,
-        name: 'Assign to Me',
+        name: 'Assign to me',
```
Comment (vikram-raj) [inline src/app/work-item/work-item.component.ts]: Your commit message is Rename filter method "Assign to Me" to "Assigned to me" but you rename it "Assign to Me" to "Assign to me"

Comment (SMahil) [inline src/app/work-item/work-item.component.ts]: @sanbornsen This has been done in my PR :)

**Comment by debloper** [comment]
Resolved as part of #891, closing this.

File: src/app/work-item/work-item.component.ts (hunk 1 above)
Comment (kwk) [inline src/app/work-item/work-item.component.ts]: @SMahil no, you have changed it to `Assigned to Me` and not `Assigned to me`. See [here](https://github.com/fabric8io/fabric8-planner/pull/891/files#diff-23889667b50a369926f3bc2f63e68311R63)

**Pull request closed without merging**